spec/controllers/relationships_controller_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'spec_helper'
+
+Rspec.describe RelationshipsController do
```

Comment:
You may leave off the initial `Rspec` here. Though, it doesn't hurt to be explicit in your method calls.